packages/babel-parser/test/fixtures/experimental/import-module-attributes/invalid-syntax-with-no-attributes-identifier/options.json

```diff
@@ -0,0 +1,8 @@
+{
+  "plugins": [
+    "moduleAttributes",
+    "estree"
+  ],
+  "sourceType": "module",
+  "throws": "Unexpected token (2:5)"
```

Comment:
Note for reviewers: this is because ASI doesn't apply, since `with` statements are disallowed in modules.

Reply:
The unexpected token error comes from `(` is not an ID_Start character. If it was parsed as a `with` statement, it should throw `Errors.StrictWith`.